model/encode/vp9.py

```diff
@@ -0,0 +1,39 @@
+###
+### Copyright (C) 2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+import os
+import slash
+
+from ...lib.ffmpeg.qsv.util import *
+from ...lib.ffmpeg.qsv.encoder import VP9_8EncoderLPTest
+from .util import TrendModelMixin
+
+spec = load_test_spec("vp9", "encode", "8bit")
+
+class trend(VP9_8EncoderLPTest, TrendModelMixin):
+  #MRO overrides
+  check_metrics = TrendModelMixin.check_metrics
+
+  def parse_psnr(self):
+    if os.path.exists(self.decoder.statsfile):
+      return parse_psnr_stats(self.decoder.statsfile, self.frames)
+
+  def initvars(self, _):
+    profile = "profile0"
+    if "AYUV" == self.format:
+      profile = "profile1"
+
+    super().initvars(profile)
+
+  @slash.parametrize(*TrendModelMixin.filter_spec(spec))
+  def test(self, case):
+    vars(self).update(case = case)
+    vars(self).update(spec[case].copy())
+    self.fit()
```

Comment:
We should use the same "modelqps" settings as we do in AV1 because VP9 has QP range 1..255.  Unfortunately, due to open bug, we'll have to modify ffmpeg-qsv to allow QP > 51 in order to generate the correct trend models.

Reply:
thanks, resolved